test/dotcom/system_status_test.exs

```diff
@@ -120,58 +169,31 @@ defmodule Dotcom.SystemStatus.AlertsTest do
 
       time = Faker.DateTime.between(start_time_2, end_time_2)
 
-      assert Alerts.active_on_day?(alert, time)
+      assert active_now_or_later_on_day?(alert, time)
     end
   end
 
-  describe "for_day/2" do
-    test "includes alerts that are active today" do
-      start_time =
-        Timex.now("America/New_York") |> Timex.beginning_of_day() |> Timex.shift(hours: 12)
-
-      end_time = Timex.now("America/New_York") |> Timex.end_of_day()
-
-      alert1 = build(:alert, active_period: [{start_time, end_time}])
+  defp disruption_alert(active_period)
+  defp disruption_alert(active_period, route_id \\ nil)
 
-      alert2 =
-        build(:alert,
-          active_period: [{start_time |> Timex.shift(days: 1), end_time |> Timex.shift(days: 1)}]
-        )
-
-      assert Alerts.for_day(
-               [alert1, alert2],
-               Faker.DateTime.between(start_time, end_time)
-             ) == [alert1]
-    end
+  defp disruption_alert(active_period, "Green") do
+    build(:alert_for_routes,
+      route_ids: GreenLine.branch_ids(),
+      active_period: [active_period],
+      effect: service_impacting_effects() |> Faker.Util.pick()
+    )
   end
 
-  describe "filter_relevant/1" do
-    test "includes an alert if its effect is :delay" do
-      alert = build(:alert, effect: :delay)
-      assert Alerts.filter_relevant([alert]) == [alert]
-    end
-
-    test "includes an alert if its effect is :shuttle" do
-      alert = build(:alert, effect: :shuttle)
-      assert Alerts.filter_relevant([alert]) == [alert]
-    end
-
-    test "includes an alert if its effect is :suspension" do
-      alert = build(:alert, effect: :suspension)
-      assert Alerts.filter_relevant([alert]) == [alert]
-    end
-
-    test "includes an alert if its effect is :station_closure" do
-      alert = build(:alert, effect: :station_closure)
-      assert Alerts.filter_relevant([alert]) == [alert]
-    end
+  defp disruption_alert(active_period, nil) do
+    route_id = Faker.Util.pick(Dotcom.Routes.subway_route_ids())
+    disruption_alert(active_period, route_id)
+  end
 
-    test "does not include alerts with other effects" do
-      assert Alerts.filter_relevant([
-               build(:alert, effect: :policy_change),
-               build(:alert, effect: :extra_service),
-               build(:alert, effect: :stop_closure)
-             ]) == []
-    end
+  defp disruption_alert(active_period, route_id) do
```

Comment:
Question: will this test file ever want to interact with alerts that are for multiple route_ids (other than the entire green line)?

Non-blocking, because this can very easily be a later question, and can be addressed if/when it's needed.

Reply:
Yeah, probably!